k8s/README.md

```diff
@@ -0,0 +1,15 @@
+# <span style="text-decoration:underline;">Deploy a carta-backend-dev Docker on Kubenete - March 2021</span>
+
+After preparing a basic Docker container of carta_backend, we apply Kubectl on creating a service on native/virtual nodes. We could aggreget many node into a working cluster. The Kubenete could easly deploy a NGINX on high availability cluster.
+
+## 1.
```

Comment:
Suppose there should be at least one 'node' in this k8s cluster, where a 'node' might be a localhost. In a mac system, we could launch a k8s environment by its reference configuration, otherwise we could use "minikube" to create this environment in a single console.
So if you have done this environment, any docker container should be able to establish on k8s.